code/game/gamemodes/clock_cult/clock_effects/clock_sigils.dm

```diff
@@ -278,7 +278,7 @@
 	animate(src, alpha = 255, time = 10, flags = ANIMATION_END_NOW) //we may have a previous animation going. finish it first, then do this one without delay.
 	sleep(10)
 //as long as they're still on the sigil and are either not a servant or they're a servant AND it has remaining vitality
-	while(L && (!is_servant_of_ratvar(L) || (is_servant_of_ratvar(L) && (GLOB.ratvar_awakens || GLOB.clockwork_vitality))) && get_turf(L) == get_turf(src))
+	while(L && (!is_servant_of_ratvar(L) || (is_servant_of_ratvar(L) && (GLOB.ratvar_awakens || GLOB.clockwork_vitality))) && get_turf(L) == get_turf(src) && !L.buckled)
```

Comment:
this should still check specifically for brass skewers, rather than any buckling. rollerchairs aren't an uncommon form of restraint.

Reply:
I thought I stated, but I guess I messed up and forgot to. Rollerchairs etc. are intended to go off, as pressure sensors detect pressure from biological creatures directly. That's also why silicons etc. didn't trigger it (I think, I may have had a different reason at the time.)